app/src/main/res/values-de/strings.xml

```diff
@@ -325,6 +325,7 @@
     <string name="sort_by_length">Nach Länge</string>
     <string name="sort_by_date_added">Nach Hinzufügungsdatum</string>
     <string name="sort_by_last_played">Zuletzt gespielt</string>
+    <string name="unsorted">Nicht Sortiert</string>
```

Comment:
"Sortiert" should be lower case: "sortiert"

Reply:
Can't you change it?